support/globalconfig/infrastructure.go

```diff
@@ -57,5 +57,8 @@ func ReconcileInfrastructure(infra *configv1.Infrastructure, hcp *hyperv1.Hosted
 	case hyperv1.IBMCloudPlatform:
 		infra.Status.PlatformStatus = &configv1.PlatformStatus{}
 		infra.Status.PlatformStatus.Type = configv1.IBMCloudPlatformType
+	case hyperv1.NonePlatform:
```

Comment:
Please just set
`infra.Status.PlatformStatus.Type = configv1.PlatformType(hcp.Spec.Platform.Type)`
for any every case, no need to discriminate by platform.